prow/entrypoint/options.go

```diff
@@ -51,6 +49,13 @@ type Options struct {
 	// May be ignored if not using sidecar.
 	ArtifactDir string `json:"artifact_dir,omitempty"`
 
+	// PreviousMarker has no effect when empty (default).
+	// When set it causes entrypoint to:
+	// a) wait until previous_marker exists
+	// b) run args as normal if previous_marker == 0
+	// c) otherwise immediately write 0 to marker_file without running args
```

Comment:
Should this be "immediately write 1126"?

Reply:
Done